src/ParallelAlgorithms/LinearSolver/ConjugateGradient/ConjugateGradient.hpp

```diff
@@ -73,107 +77,21 @@ struct ConjugateGradient {
   using component_list = tmpl::list<
       detail::ResidualMonitor<Metavariables, FieldsTag, OptionsGroup>>;
 
-  /*!
-   * \brief Initialize the tags used by the conjugate gradient linear solver.
-   *
-   * Since we have not started iterating yet, we initialize the state _before_
-   * the first iteration. So `LinearSolver::Tags::IterationId` is undefined at
-   * this point and `Tags::Next<LinearSolver::Tags::IterationId>` is the initial
-   * step number. Invoke `prepare_step` to advance the state to the first
-   * iteration.
-   *
-   * \warning This action involves a blocking reduction, so it is a global
-   * synchronization point.
-   *
-   * With:
-   * - `operand_tag` =
-   * `db::add_tag_prefix<LinearSolver::Tags::Operand, FieldsTag>`
-   * - `operator_tag` =
-   * `db::add_tag_prefix<LinearSolver::Tags::OperatorAppliedTo, operand_tag>`
-   * - `residual_tag` =
-   * `db::add_tag_prefix<LinearSolver::Tags::Residual, FieldsTag>`
-   *
-   * DataBox changes:
-   * - Adds:
-   *   * `LinearSolver::Tags::IterationId`
-   *   * `Tags::Next<LinearSolver::Tags::IterationId>`
-   *   * `residual_tag`
-   *   * `LinearSolver::Tags::HasConverged`
-   * - Removes: nothing
-   * - Modifies:
-   *   * `operand_tag`
-   *
-   * \note The `operand_tag` must already be present in the DataBox and is set
-   * to its initial value here. It is typically added to the DataBox by the
-   * system, which uses it to compute the `operator_tag` in each step. Also the
-   * `operator_tag` is typically added to the DataBox by the system, but does
-   * not need to be initialized until it is computed for the first time in the
-   * first step of the algorithm.
-   */
   using initialize_element = detail::InitializeElement<FieldsTag, OptionsGroup>;
 
   using register_element = tmpl::list<>;
 
-  /*!
```

Comment:
The documentation I deleted was stale or just wrong because it referred to implementation details that changed. I don't think it was ever useful. I'll probably add a dev guide on how to run linear solves at some point instead. It could also help to add a protocol that documents the interface that all linear solvers have in common (they have `initialize_element`, `register_element` and `solve` action lists).